.circleci/config.yml

```diff
@@ -1937,9 +2016,19 @@ workflows:
           name: "PHP 80 Xdebug tests"
           docker_image: "datadog/dd-trace-ci:php-8.0_buster"
           xdebug_version_one: "3.0.0"
+      - xdebug_tests:
+          requires: [ 'Prepare Code' ]
+          name: "PHP 81 Xdebug tests"
+          docker_image: "datadog/dd-trace-ci:php-8.1_buster"
+          xdebug_version_one: "3.1.0"
       - placeholder:
           requires: [ 'Prepare Code' ]
           name: Language tests
+      - php_language_tests:
+          requires: [ 'Language tests' ]
+          name: "PHP 81 language tests"
+          xfail_list: dockerfiles/ci/xfail_tests/8.1.list
+          docker_image: "datadog/dd-trace-ci:php-8.1_buster"
       - php_language_tests:
           requires: [ 'Language tests' ]
           name: "PHP 80 language tests"
```

Comment:
Commenting here as this is the last line opened for https://github.com/DataDog/dd-trace-php/pull/1297/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R2077

Is it expected to miss the equivalent of `PHP 80 curl integration tests a shared lib` for 8.1?

Reply:
We only have a single version compiled with shared lib, so I assume we'll at some time (after release) replace the shared 80 by shared 81.

Reply:
👍🏻  sounds good. Please create a Jira so we do not forget.

Reply:
https://datadoghq.atlassian.net/browse/APMPHP-632 done